### ISSUE TITLE
SV genotyping

### ISSUE THREAD

**Issue by zhiquanyang** [fritzsedlazeck/Sniffles#175]
Hello! Did you compare Sniffles with LRcaller about SV genotyping? What's the difference the two softwares in methods?

**Comment by fritzsedlazeck** [comment]
I have not. I don't think I heared from lrcaller before.

**Comment by zhiquanyang** [comment]
:)The paper is here (http://dx.doi.org/10.1101/848366). The author used Sniffles to call SVs and LRcaller to do genotyping

**Comment by fritzsedlazeck** [comment]
Ah sorry yes saw that paper. They require illumina data and also go back to the raw signal... So sniffles doenst need either.
But no I haven't compared these and they haven't either :).
Thanks
Fritz

**Comment by zhiquanyang** [comment]
Thank you very much! Sorry but I have another important question. We just want to use long reads SV sets to do genotyping for short reads in population. Do you have any recommended packages for that?

**Comment by fritzsedlazeck** [comment]
Oh I see. Then neither would work. Sniffles genotyper is more for harmonizing a long read population set, whereas theirs is more for filtering I belive. Anyways for short reads:

1. I published a benchmark paper this year: https://academic.oup.com/gigascience/article/8/9/giz110/5565134 
2. Also I think I mentioned some methods briefly in the review paper:
https://genomebiology.biomedcentral.com/articles/10.1186/s13059-019-1828-7

3. This paper is about utilizing a graph genome structure, but its a bit sensitive to the breakpoint precision. We developed that together with Illumina. It is already accepted and will soon come out: https://www.biorxiv.org/content/10.1101/635011v1  

4. And here is one that is a bit outside of the box, but works nicely. We are in the process to write that up with Ryan: https://github.com/ryanlayer/stix 

I hope that helps. Feel free to reach out if you have questions: fritz.sedlazeck@gmail.com

Best
Fritz
